assets/js/app/LocusZoom.js

```diff
@@ -48,7 +59,7 @@ LocusZoom.populate = function(selector, datasource, layout, state) {
 LocusZoom.populateAll = function(selector, datasource, layout, state) {
     var instances = [];
     d3.selectAll(selector).each(function(d,i) {
-        instances[i] = LocusZoom.populate(this, datasource, layout, state);
+        instances[i] = LocusZoom.populate(this, datasource, layout);
```

Comment:
State should be passed through here as well. Perhaps add a test for this behavior? Maybe throw a warnings that this parameter is being deprecated?